index.js

```diff
@@ -8,8 +8,8 @@ Argv(process.argv.slice(2))
 
 module.exports = Argv
 
-function Argv (processArgs, cwd) {
-  const argv = yargs(processArgs, cwd, require)
+function Argv (processArgs, cwd, parentRequire) {
```

Comment:
I'm thinking maybe we make the second parameter take an object:

```js
{cwd: 'foo', require: func () {}}
```

if a string is provided we'd fallback to the `cwd` behavior.